css/custom-responsive.css

```
@@ -1,32 +1,67 @@
 :root {
     --columns: 12;
-    --gutter: 32px;
+    --gap: 32px;
     --sm-breakpoint: 576px;
     --md-breakpoint: 768px;
     --lg-breakpoint: 992px;
     --xl-breakpoint: 1200px;
 }
 
 .container {
-    width: 100%;
-    padding-right: calc(var(--gutter) / 2);
-    padding-left: calc(var(--gutter) / 2);
-    margin-right: auto;
-    margin-left: auto;
+  width: 85%;
+  margin: 0 auto;
+}
+
+@media (min-width: 576px) {
+  .container {
+      max-width: 95%;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+      max-width: 90%;
+  }
+}
+
+@media (min-width: 992px) {
+  .container {
+      max-width: 85%;
+  }
+}
+
+@media (min-width: 1200px) {
+  .container {
+      max-width: 80%;
   }
+}
+
+@media (min-width: 1400px) {
+  .container {
+      max-width: 75%;
+  }
+}
+
+
+.container-fluid {
+  width: 100%;
+  margin-right: auto;
+  margin-left: auto;
+}
+
```

full width thì không có margin nữa
```suggestion
.container-fluid {
  width: 100%;
}

```